tests/model_monitoring/test_tdengine.py

```diff
@@ -186,16 +186,16 @@ def test_get_subtables(
                 "subtable_1",
                 [],
                 "",
-                datetime.datetime.now().astimezone() - datetime.timedelta(hours=1),
-                datetime.datetime.now().astimezone(),
+                mlrun.utils.datetime_now() - datetime.timedelta(hours=1),
```

Comment:
Here `astimezone()` is removed, but new lines below are added with `datetime.datetime.now().astimezone()`. Does this inconsistency make sense? Do we need the local time zone or naive timestamps?

Reply:
yeah it doesn't really matter in that test scenario but as you mentioned I'll remove `astimezone()` from the new lines as well to keep consistency